jepsen/test/jepsen/util_test.clj

```diff
@@ -78,11 +77,11 @@
 
 (deftest letr-test
   (testing "no bindings"
-    (is (= (letr []) nil))
-    (is (= (letr [] 1 2) 2)))
```

Comment:
This is another case where I feel like the linter is being overzealous. We *want* this intermediate form to go unused! That's what we're testing for. :-)

Reply:
I'll try a bit more to disable lint rules on a per function basis.

Reply:
Fixed.